blog/posts/swiip/the-weight-of-js.md

```diff
@@ -0,0 +1,191 @@
+---
+lang: en
+title: The Weight of JS
+description: A deep dive on why the weight of your JavaScript code does matter.
```

Comment:
The description should be longer to help user find your blog post.
Imagine you are a random internet user which is looking for what you are explaining in here, what word will he use ?